src/twisted/internet/unix.py

```diff
@@ -294,6 +294,13 @@ def getHost(self):
     def getPeer(self):
         return address.UNIXAddress(self.hostname or None)
 
+    def getTcpNoDelay(self):
+        return False
+
+    def setTcpNoDelay(self, enabled):
+        # This is not supported on UNIX sockets and therefore silently ignored.
```

Comment:
I tnink that we should reference the followup ticket

This is kind of a workaround for our bad implementantation of `twisted.internet.unix.Server` which should not declare that it implements the ITCPTransport inteface

```suggestion
        """        FIXME:https://github.com/twisted/twisted/issues/12369
        
        This is not supported on UNIX sockets and therefore silently ignored.
        """
```

Reply:
Fixed, thanks!